doc/user_guide/tools.rst

```diff
@@ -62,12 +64,12 @@ information (including calibration) can be accessed (and modified) in the
 Transforming between signal subclasses
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The different subclasses are characterized by three
+The different subclasses are characterized by four
 :py:attr:`~.signal.BaseSignal.metadata` attributes (see the table below):
 
 `record_by`
     Can be "spectrum", "image" or "", the latter meaning undefined and describes
-    the way the data is arranged in memory. It is possible to transform any
+    the way the data is arranged in memory. It is possible to transform any non-complex
```

Comment:
I think this was left over from before we had distinct classes for `ComplexSignal1D/2D`...
It's removed now.